src/main/resources/default/assets/javascript/multiLanguageField.js

```diff
@@ -19,6 +19,7 @@ function MultiLanguageField(options) {
     this.languageManagementEnabled = options.languageManagementEnabled;
     this.rows = options.rows;
     this.multiline = options.multiline || false;
+    this.mobileOrSmallScreen = window.matchMedia('(max-width: 600px)').matches;
```

Comment:
shouldnt we put such methods into a central place like the core.js in sirius-web - if we ever change page breaks or the like no one will ever find this...

Reply:
A better approach instead of only checking the viewport width would be to check for the label length and the tab width of all visible tabs in combination. Therefore more than 3 tabs could be displayed depending on the screen size. (thx @mbrsci for the idea)